Rakefile

```diff
@@ -177,19 +177,19 @@ end
 
 desc "Run all code generation tasks"
 task :generate do
```

Comment:
Unrelated to this PR, but I just noticed this task should depend on `:dependencies` since it uses a command from `Pods`